easybuild/easyconfigs/l/LSMS/LSMS-Makefiles.patch

```diff
@@ -0,0 +1,106 @@
+--- LSMS_3_rev237/CBLAS/Makefile	2012-02-09 20:17:00.000000000 +0100
```

Comment:
@foxik0070 please include a description of the patch + link to where it comes from and/or author